modules/ROOT/pages/cassandra/cassandra-config-topics.adoc

```diff
@@ -0,0 +1,27 @@
+= Configuration Topics - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Cassandra DB connector configuration must match the Cassandra configuration with regard to Basic Authentication.
```

Comment:
You Initial cap Cassandra Connector later on. We need to add some rules for when to initial cap Connector and when not to,

Reply:
Were you able to check the Salesforce Style Guide for rules regarding this item? If it wasn't there, do you want to start a list of questions?

Reply:
Fixed to be consistent.